tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/metrics/util/MetricComparator.java

```diff
@@ -35,10 +36,12 @@ private MetricComparator() {}
      * <p>
      * Useful for checking the results from Histograms.
      * 
-     * @param millis the expected time in milliseconds
+     * @param originalMillis the expected time in milliseconds
      * @return a {@link Matcher} which matches against a time in milliseconds
      */
-    public static Matcher<Double> approxMillis(long millis) {
+    public static Matcher<Double> approxMillis(final long originalMillis) {
+
+        final long millis = TCKConfig.getConfig().getTimeoutInMillis(originalMillis);
```

Comment:
We should also scale the margin here as well.

Reply:
Hi @Azquelt, I don't know if I follow. What do you mean by the margin?

Reply:
Sorry, my comment was really unclear :(

`closeTo` takes two parameters, the expected value and the allowed margin of error. I think we should be scaling both values rather than just one.

Possibly we should even change this method so that rather than having a fixed margin of error of +/- 100ms, it uses something like +/- 20% of the expected value.

Reply:
Thanks Andrew. I'm going to do the change

Reply:
done